concepts/custom.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Customize theme
+---
+
+Bored with basic theme? Here is the solution to customize your app!
+You can change the default colors of the app in your `config.toml`.
+
+## Change the light theme
+
+To customize the light theme, find this section who start `[UI.theme.light]` and update the color you want. If you don't find it, just copy/past it on `config.toml`.
```

Comment:
i would say add or edit since it might not exist in config generated before this feature was released